src/lib/db/migrations/20230310111029_initial.ts

```diff
@@ -24,6 +24,11 @@ export async function up(knex: Knex): Promise<void> {
     def.integer('embodied_co2').nullable().index('embodied_co2_index').defaultTo(null)
     def.string('energy_owner', 48).notNullable()
     def.string('hydrogen_owner', 48).notNullable()
+    def.dateTime('production_start_time').nullable().defaultTo(null)
```

Comment:
Out of interest, do `dateTime` outputs match the dateTime format of the carbon intensity API (https://carbon-intensity.github.io/api-definitions/#get-intensity-from-to)